be/src/common/config.h

```diff
@@ -982,7 +982,7 @@ CONF_mInt64(lake_pk_compaction_min_input_segments, "5");
 // Used for control memory usage of update state cache and compaction state cache
 CONF_mInt32(lake_pk_preload_memory_limit_percent, "30");
 CONF_mInt32(lake_pk_index_sst_min_compaction_versions, "2");
-CONF_mInt32(lake_pk_index_sst_max_compaction_versions, "5");
+CONF_mInt32(lake_pk_index_sst_max_compaction_bytes, /*1GB*/ "1073741824");
```

Comment:
can it be done as "compaction up to X bytes with at most Y files at a time"?

Reply:
do you mean `lake_pk_index_sst_max_compaction_versions` and `lake_pk_index_sst_max_compaction_bytes` should also be needed?

Reply:
not sure if there is something we can learn from the funplus compaction issue. when the memory is limited, can the compaction be still making progress in small steps. Or can this be automatically adjusted based on memory back pressure?

Reply:
`lake_pk_index_sst_max_compaction_bytes` can also be used to limit the memory.